targets/custom_targets.json

```diff
@@ -0,0 +1,65 @@
+{
+    "ARM_REV2": {
+        "inherits": [
+            "UWRT_STM32F446xE"
+        ],
+        "overrides": {
+            "clock_source": "USE_PLL_HSE_XTAL|USE_PLL_HSI",
+            "hse_value": "25000000U"
```

Comment:
this replaces the hal config header we needed before. all the other hal flags are alread configured by the stm32 provided cmakelists.txt